kafka-ui-react-app/src/lib/hooks/api/kafkaConnect.ts

```diff
@@ -51,11 +51,13 @@ export function useConnects(clusterName: ClusterName) {
   );
 }
 export function useConnectors(clusterName: ClusterName, search?: string) {
+  const client = useQueryClient();
   return useQuery(
     connectorsKey(clusterName, search),
     () => api.getAllConnectors({ clusterName, search }),
     {
       select: (data) => sortBy(data, 'name'),
+      onSuccess: () => client.invalidateQueries(),
```

Comment:
this will invalidate everything and everything , if we can do more specific we need to specify the query keys.

Reply:
removed invalidateQueries from useConnectors and set it in useUpdateConnectorState with query keys